GestionareStructuraEchipaDeFormula1/src/gestionareStructuraEchipaDeFormula1/Servicii.java

```diff
@@ -0,0 +1,108 @@
+package gestionareStructuraEchipaDeFormula1;
+
+import gestionareStructuraEchipaDeFormula1.Angajat.AltJob;
+import gestionareStructuraEchipaDeFormula1.Angajat.Angajat;
+import gestionareStructuraEchipaDeFormula1.Angajat.Pilot;
+import gestionareStructuraEchipaDeFormula1.Angajat.StaffCursa;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Servicii {
```

Comment:
you can try to split this service into multiple services - ServiciuPilot, ServiciuSaff, ServiciuMagazin, etc - in those services you will keep just the logic specific to that entity